tests/thread_local_large/enc/enc.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/tests.h>
+#include <array>
+#include <cstdint>
+#include "thread_local_large_t.h"
+
+using namespace std;
+
+void enc_test()
+{
+    static thread_local array<uint8_t, 9000> a;
+    for (auto& x : a)
+    {
+        OE_TEST(x == 0);
+        x = 2;
+    }
+    host_nop();
```

Comment:
What is the purpose of `host_nop`?

Reply:
This should make sure that TLS is fine after an ocall. I'll add a comment.